### ISSUE TITLE
Creating new model based on specific source model fails

### ISSUE THREAD

**Issue by artgadliba** [d8ahazard/sd_dreambooth_extension#585]
**Please find the following lines in the console and paste them below. _If you do not provide this information, your issue will be automatically closed._**

#######################################################################################################
Initializing Dreambooth
If submitting an issue on github, please provide the below text for debugging purposes:

Python revision: 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Dreambooth revision: c678a431b5d79ba7e1c38d7629ca7e0c79166a1f
SD-WebUI revision: 685f9631b56ff8bd43bce24ff5ce0f9a0e9af490

Checking Dreambooth requirements...
[+] bitsandbytes version 0.35.0 installed.
[+] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] xformers version 0.0.14.dev0 installed.
[+] torch version 1.12.1+cu113 installed.
[+] torchvision version 0.13.1+cu113 installed.
#######################################################################################################

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes
**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes
**Have you updated Dreambooth to the latest revision?**
Yes
**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Yes
**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this. Reply 'OK' Below to acknowledge that you did this.**
Ok
**Describe the bug**
Creating new model with sd-v1-5-inpainting.ckpt source fails. What could be wrong? Other source files works fine

(A clear and concise description of what the bug is)

**Provide logs**

Launching Web UI with arguments: --xformers --test-lora
Dreambooth API layer loaded
LatentInpaintDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.54 M params.
Loading weights [3e16efc8] from C:\stable-diffusion-webui\models\Stable-diffusion\sd-v1-5-inpainting.ckpt
Applying xformers cross attention optimization.
Model loaded.
Loaded a total of 0 textual inversion embeddings.
Embeddings:
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Loading model from checkpoint.
Loading checkpoint...
v1 model loaded.
Creating scheduler...
Converting unet...
Pipeline or config is not set, unable to continue.
 Restored system models.
 Allocated: 2.0GB
 Reserved: 2.0GB

 Allocated 2.0/2.0GB
 Reserved: 2.0/2.0GB

Exception while extracting model: Error(s) in loading state_dict for UNet2DConditionModel:
        size mismatch for conv_in.weight: copying a param with shape torch.Size([320, 9, 3, 3]) from checkpoint, the shape in current model is torch.Size([320, 4, 3, 3]).

If a crash has occurred, please provide the *entire* stack trace from the log, including the last few log messages *before* the crash occurred.

**Environment**

Windows Server 2022
GTX 3090 

**Screenshots/Config**
If the issue is specific to an error while training, please provide a screenshot of training parameters or the db_config.json file from /models/dreambooth/MODELNAME/db_config.json

**Comment by d8ahazard** [comment]
I'm aware Shivam has inpainting support. However, it's a whole new script
to add, and another can of worms to open.

I do intend on adding it at some point, but first I just want to
ensure everything else I have implemented is working properly. ;)
